yenly/smart-shopping-list-typed
# Feat Connect firebase

## Description
- Installed `firebase` and `react-firebase-hooks` packages and set up this app to connect to firebase.
- Created example `BoopFirebase` component to add data to and read data from firebase.

## Acceptance Criteria


## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |




## Testing Steps / QA Criteria

1. run app locally
2. click boop button
3. check to see  a new boop appear on list below.
4. log in to https://console.firebase.google.com,
5. under boops collection, create a new document with new boop
6. check your app running on localhost to see new boop
